IBAnimatableTests/ColorTypeTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ColorTypeTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/26/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
```

Comment:
Can we remove the @testable? It would help to ensure that we don't forget a public